01-main/packages/neohtop

```diff
@@ -0,0 +1,10 @@
+DEFVER=1
+ARCHS_SUPPORTED="x86_64 aarch64"
+get_github_releases "Abdenasser/neohtop" "latest"
+if [ "${ACTION}" != "prettylist" ]; then
+    URL="$(grep "browser_download_url.*${HOST_ARCH}\.deb\"" "${CACHE_FILE}" | grep -v '.md5' | head -n1 | cut -d'"' -f4)"
```

Comment:
ar```suggestion
    URL=$(grep -m1 "browser_download_url.*${ARCH}\.deb\"" "${CACHE_FILE}" | cut -d'"' -f4)
```
They are using ARCH rather than HOST_ARCH